underfs/local/src/main/java/tachyon/underfs/local/LocalUnderFileSystemFactory.java

```diff
@@ -38,6 +38,8 @@ public boolean supportsPath(String path, TachyonConf tachyonConf) {
     if (path == null) {
       return false;
     }
-    return path.startsWith(TachyonURI.SEPARATOR) || path.startsWith("file://");
+    return path.startsWith(TachyonURI.SEPARATOR)
+        || path.startsWith("file://")
+        || TachyonURI.hasWindowsDrive(path, false);
```

Comment:
Whether a path is a Windows style path should not be platform dependent. Surely we barely meet a Windows style path in *nix platform. But I think we don't need to add such restrictions. I removed this restriction in `TachyonURI.hasWindowsDrive` as well, because it failed some UTs in Linux platform which should pass both on Windows & Linux.